docs/en/modules/net.md

```diff
@@ -282,7 +338,7 @@ end)
 
 ## net.socket:unhold()
 
-Unblock TCP receiving data, i.e. undo `hold()`.
+Unblock TCP receiving data. I.e. undo `hold()`.
```

Comment:
I see...never noticed, thanks. My JavaScript that generates the table just reads to the first '.' assuming that it terminates the first sentence :frowning:. I see two quick fixes:
- find a dot for 'i.e.' that is not the regular dot
- replace the 'i.e.' phrase with one of its English synonyms: https://en.wikipedia.org/wiki/List_of_Latin_phrases_(I)#id_est